snow/engine/snowman/bootstrap/block_job.go

```diff
@@ -66,8 +66,8 @@ func (b *blockJob) Execute() error {
 		return fmt.Errorf("attempting to execute block with status %s", status)
 	case choices.Processing:
 		if err := b.blk.Verify(); err != nil {
-			b.log.Debug("block %s failed verification during bootstrapping due to %s",
-				b.blk.ID(), err)
+			return fmt.Errorf("block %s failed verification during bootstrapping due to %s",
+				b.blk.ID(), err, b.blk)
```

Comment:
I think you included `b.blk` by accident

Reply:
You're right. This has already been placed into the release. So I'll address this in a future PR